index.html

```diff
@@ -177,6 +177,24 @@ <h1>The Registry</h1>
   </thead>
 
   <tbody>
+    <tr>
+      <td>
+          did:abt:
+      </td>
+      <td>
+          PROVISIONAL
+      </td>
+      <td>
+          ABT Network
+      </td>
+      <td>
+          Robert Mao, Tyr Chen, Peiling Ding
+      </td>
+      <td>
+          <a href="https://github.com/ArcBlock/ABT-DID-Protocol/blob/master/README.md">ABT DID Method</a>
```

Comment:
create a git website to it or serve our repo as a website, don't point to the file. 

reference how lifeid and others did.